src/MathOptInterface.jl

```diff
@@ -124,23 +124,41 @@ This is used in two places to determine whether to add a cache:
 supports_incremental_interface(::ModelLike, ::Bool) = false
 
 """
-    copy_to(dest::ModelLike, src::ModelLike; copy_names=true, warn_attributes=true)
-
-Copy the model from `src` into `dest`. The target `dest` is emptied, and all
-previous indices to variables or constraints in `dest` are invalidated. Returns
-a dictionary-like object that translates variable and constraint indices from
-the `src` model to the corresponding indices in the `dest` model.
-
-If `copy_names` is `false`, the `Name`, `VariableName` and `ConstraintName`
-attributes are not copied even if they are set in `src`. If a constraint that
-is copied from `src` is not supported by `dest` then an
-[`UnsupportedConstraint`](@ref) error is thrown. Similarly, if a model, variable
-or constraint attribute that is copied from `src` is not supported by `dest`
-then an [`UnsupportedAttribute`](@ref) error is thrown. Unsupported *optimizer*
-attributes are treated differently:
-
-* If `warn_attributes` is `true`, a warning is displayed, otherwise,
-* the attribute is silently ignored.
+    copy_to(
+        dest::ModelLike,
+        src::ModelLike;
+        copy_names::Bool = true,
```

Comment:
Remind me why we need `copy_names`? Why can't we just try by default, and skip if they aren't supported?